### DISCUSSION TITLE
Mobile Apps

### DISCUSSION THREAD

**Discussion by 1beb** [bigskysoftware/htmx#1802] [Q&A]
Hello Community, 

Is there anyone out there using HTMX in a Hybrid Web and Mobile App? If so, can you discuss your stack/strategy?

**Comment by nickchomey** [comment]
Do you mean a native app or pwa? If the latter, I intend to do so and have written about it in the pwa-corner channel of the discord. Others there are interested as well.

**Reply by 1beb** [reply]
For example, if I wanted to build a hybrid app today I would create an API and a separate front-end using a js framework, wrapping it with CapacitorJS to get "web native" apps that I could post to app stores for Android and iOS. 

My understanding is that a PWA does not create installable applications  for the stores, unless I'm mistaken?

**Reply by nickchomey** [reply]
You could check out https://www.pwabuilder.com/

Also, there's Hyperview, which is another way to build hypermedia mobile apps. You can read about it in the Hypermedia.systems book
https://hypermedia.systems/hyperview-a-mobile-hypermedia

Id also reflect on why you even want an app in the app stores. PWAs have come so far, that I truly can't think of much reason to want one - you need to maintain a separate codebase, pay exorbitant fees to the stores etc...
